src/main/java/tuteez/logic/Messages.java

```diff
@@ -14,6 +14,7 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_NEED_TO_PROVIDE_INDEX = "You must provide an index";
```

Comment:
Currently, addrmk and delrmk is using `MESSAGE_MISSING_PERSON_INDEX`, perhaps you could use this instead so that we can reduce code duplication